modules/database/src/ioc/rsrv/caservertask.c

```diff
@@ -590,7 +590,7 @@ void rsrv_init (void)
             errlogPrintf ( "cas " ERL_WARNING ": reachable with UDP unicast (a host's IP in EPICS_CA_ADDR_LIST)\n" );
         }
 
-        epicsSnprintf(buf, sizeof(buf)-1u, "%u", ca_server_port);
+        epicsSnprintf(buf, sizeof(buf), "%u", ca_server_port);
         buf[sizeof(buf)-1u] = '\0';
```

Comment:
I guess my `-1` paranoia is unnecessary.  So the following line should be removed.

Reply:
done